keps/sig-network/20190125-ingress-api-group.md

```diff
@@ -493,6 +505,7 @@ backend:
 ### 1.16
 
 * [ ] Meet graduation criteria and promote API to `networking.k8s.io/v1`
+* [ ] Implement API changes to GA version.
```

Comment:
Honestly, in my opinion we should not change the current API contracts we have right now. 

- HealthChecks provide a huge change. Healthchecks should be defined on the pool member object, like endpoints and are already defined in the container spec. See also traditional LBs like F5 define these on the pool member. Also we don't need active healthchecks in Kubernetes ingress at all, because of the current endpoints behavior!
- If the Hostname proposal means having .TLD as wildcard possible like `example.*`, no DNS automation can work, which breaks for hundreds of production workloads the possible automation. As far as I understand this is not the case, but it should be explicitly stated.
- Path changes might create major impact on current running systems. I think sig-network should also care about migrations, because ingress is one of the biggest impact objects for all production clusters running in the world. 

Please take it very serious, that there are hundreds or thousands of production clusters out there that might get hit by incompatible changes. I would like to have these in a v2, instead of the move to GA.

Reply:
- We've moved punted on the healthchecks for now.
- TLD as wildcard is not possible (description has been clarified).
- Path should be roundtrip'able as defined

In general, as the version changes (even for v1beta3), we would expect some amount of migration to happen.

Reply:
Recomment if you still have open questions. It is required that the GA resource be roundtrippable (for existing beta features) with the Beta resource.